### PR TITLE
Comment Author Name and Time

### PR DESCRIPTION
**Pull request by pperum002c**
Just updated in comments section, additional field details like author name and commented time.

### CODE REVIEW

**Review by pperum002c — COMMENTED** [review]
Added Comment Attributes

File: docs/examples.rst
Comment (studioj) [inline docs/examples.rst:254]:
```suggestion
Get comment author name and comment creation timestamp if you know its ID::
```

**Review by studioj — CHANGES_REQUESTED** [review]
looks like the maintainers want to have the ci system back up and running before agreeing to any new merges.

if you want to contribute to get yr pr through have a look here => https://github.com/pycontribs/jira/issues/896

Maybe since its documentation only they can have it merged.
BTW I'm not a maintainer :-)

**Review by ssbarnea — APPROVED** [review]

**Review by studioj — APPROVED** [review]

**Comment by studioj** [comment]
@ssbarnea  seems this can be merged

**Review by ssbarnea — APPROVED** [review]

**Pull request merged**
